pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -209,6 +213,9 @@ func (r *ReconcileComplianceScan) validate(instance *compv1alpha1.ComplianceScan
 	// If no phase set, default to pending (the initial phase):
 	if instance.Status.Phase == "" {
 		instanceCopy := instance.DeepCopy()
+		if instance.Status.RemainingRetries != instance.Spec.MaxRetryOnTimeout {
```

Comment:
Do we need this if? Wouldn't this work equally well if we just set `instanceCopy.Status.RemainingRetries = instance.Spec.MaxRetryOnTimeout` ?

Reply:
Yes, it would be the same